packages/components/ng-at-internet-ui-router-plugin/package.json

```diff
@@ -42,4 +42,4 @@
     "@uirouter/angularjs": "^1.0.22",
     "angular": "~1.6"
   }
-}
\ No newline at end of file
+}
```

Comment:
This shouldn't be brought by this PR

Reply:
Thanks for pointing out this issue.

Branch `feat/contacts-ca` has been rebased on `develop` so this is now fixed.